Adding Note regarding the pressure capabilities of the sensor. Changing the use of busio. Including learning guide. (#47)
adafruit/Adafruit_CircuitPython_BME280: README.rst

    import board
    import digitalio
    import time
    import adafruit_bme280

>     # Create library object using our Bus I2C port

Reviewer: ```suggestion
    # Create sensor object, using the board's default I2C bus.
```

    i2c = board.I2C()   # uses board.SCL and board.SDA
    bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
    #or with other sensor address
    #bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c, address=0x76)

>     # OR create library object using our Bus SPI port

Reviewer: ```suggestion
    # OR create sensor object, using the board's default SPI bus.
```
